scripts/van_allen_tools/migrate_van_allen_workspaces.py

```diff
@@ -131,68 +128,214 @@ def make_create_workspace_request(workspace_name, auth_domains, namespace=NAMESP
     return create_ws_request
 
 
+def copy_workspaces_workflows(destination_workspace_namespace, destination_workspace_name, source_workspace_namespace, source_workspace_name):
+    """Copy Van Allen Lab workspaces Workflows."""
+    # Get the list of all the workflows
+    try:
+        workflow_list = fapi.list_workspace_configs(source_workspace_namespace, source_workspace_name)
+
+        # Store all the workflow names
+        workflow_name_list = []
+
+        for workflow in workflow_list.json():
+            # Get workflow config (overview config)
+            workflow_config = workflow['methodRepoMethod']
+
+            workflow_name_list.append(workflow_config['methodName'])
+
+            # Get workspace config (Detailed config with inputs, oututs, etc)
+            workspace_config = fapi.get_workspace_config(source_workspace_namespace, source_workspace_name, workflow_config['methodNamespace'], workflow_config['methodName'])
+
+            # Create a workflow based on Detailed config
+            fapi.create_workspace_config(destination_workspace_namespace, destination_workspace_name, workspace_config.json())
+            print(f"Copied {workflow_config['methodName']} workflow : over to {destination_workspace_namespace}/{destination_workspace_name}")
+
+        # Check if workflows match
+        new_workflow_list = fapi.list_workspace_configs(destination_workspace_namespace, destination_workspace_name).json()
+        if workflow_list != workflow_list:
+            print(f"Error: Data Tables don't match")
+            return False, f"Error: Data Tables don't match", None
```

Comment:
maybe instead of returning 3 things we can simplify it here and do 
```
missing_workflow_names = list(set(source_workflow_names) - set(destination_workflow_names))
error = f"Not all workflow from {source_workspace_name} copied over to {destination_workspace_name}. List of missing workflows: {missing_workflow_names}"

return False, error
```